packages/lesmis-server/src/connections/WebConfigConnection.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+import { ApiConnection, QueryParameters } from '@tupaia/server-boilerplate';
+const { WEB_CONFIG_API_URL = 'http://localhost:8000/api/v1' } = process.env;
+
+export class WebConfigConnection extends ApiConnection {
+  baseUrl = WEB_CONFIG_API_URL;
+
+  async getDashboardReport(query: QueryParameters) {
```

Comment:
We have this semi-convention of using `fetch` when a getter is async.
```suggestion
  async fetchDashboardReport(query: QueryParameters) {
```
(same below)

Reply:
Ah I was curious why we were inconsistent about the language around fetch/get, that makes sense